### PR TITLE
Prepare release v3.2.0

### PR DESCRIPTION
**Pull request by ACascarino**
* UPDATED: Tested against fwk_io v3.5.1 updated from v3.3.0
  * FIXED: Windows host issue with pre-2013 Visual Studio Compiler and stdbool.h.
  * UPDATED: libusb host drivers for Windows to v1.0.27. The device must now use WinUSB drivers on Windows.
  * UPDATED: To tinyusb_src commit c61f5f4, it includes official support for USB test mode.
  * FIXED: Adopted common format in error printouts of device control host drivers.

### CODE REVIEW

**Review by mbanth — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 RTOS Framework change log
 =========================
 
+UNRELEASED
```
[the remark this answers is not in the document]
Comment (ACascarino) [reply CHANGELOG.rst]: Resolved in https://github.com/xmos/fwk_rtos/pull/241/commits/1133566d2215fd285ffe1c9114bdb3e4417d85b1

File: Jenkinsfile [hunk 2]
```diff
@@ -1,5 +1,27 @@
 @Library('xmos_jenkins_shared_library@v0.28.0') _
```
[the remark this answers is not in the document]
Comment (ACascarino) [reply Jenkinsfile:1]: As agreed in backlog grooming, will also bump Tools to 15.3.0

File: modules/sw_services/device_control/host/device_access_usb.c [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
-#include <libusb.h>
 #endif
+#include <libusb.h>
 #include "device_control_host.h"
 #include "control_host_support.h"
 #include "util.h"
 
 //#define DBG(x) x
 #define DBG(x)
+#define PRINT_ERROR(...)   fprintf(stderr, "Error  : " __VA_ARGS__)
```
[the remark this answers is not in the document]
Comment (ACascarino) [reply modules/sw_services/device_control/host/device_access_usb.c:23]: Agreed. This definition appears to only be used in the device_control sw_service so will put it in a common area for those. Another declaration for it exists in argtable.c but that's a different definition so will leave that alone.

**Review by mbanth — APPROVED** [review]
I have made a few comments.  They all mention optional changes, nothing that I think is required as part of the release.

**Comment by ACascarino** [comment]
@mbanth thank you for your comments - I'd like to raise stories in LSM to address these in a future release

**Pull request merged**